### DISCUSSION TITLE
NodeJS API vs Python

### DISCUSSION THREAD

**Discussion by rayishome** [duckdb/duckdb#7057] [General]
It feels like most of the work is targeted towards Python?  For example there are features like support of Azure Storage Account streaming in Python and other features like this not present in the Node api.

**Comment by Mytherin** [comment]
Almost all of the work done on DuckDB is done in the core system which is available to all clients. The Python client does not specifically support Azure Storage Account streaming - but rather the Python eco-system provides a generic filesystem layer that DuckDB integrates with, and other libraries then provide support for various back-ends through this integration. If a similar layer is available in the NodeJS world feel free to open a feature request.

**Reply by rayishome** [reply]
I created a feature request here: https://github.com/duckdb/duckdb/discussions/7063
